omod/src/test/resources/org/openmrs/module/idgen/include/TestData.xml

```diff
@@ -0,0 +1,38 @@
+	<IDGEN_IDENTIFIER_SOURCE ID="2" UUID="0d47284f-9e9b-4a81-a88b-8bb42bc0a902" NAME="Test Remote Source" IDENTIFIER_TYPE="2" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
+	<IDGEN_IDENTIFIER_SOURCE ID="3" UUID="0d47284f-9e9b-4a81-a88b-8bb42bc0a903" NAME="Test Identifier Pool" IDENTIFIER_TYPE="1" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
+    <IDGEN_IDENTIFIER_SOURCE ID="4" UUID="0d42284f-9e9b-4a81-a88b-8bb42bc0a904" NAME="Another Test Identifier Pool" IDENTIFIER_TYPE="4" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
+	<IDGEN_IDENTIFIER_SOURCE ID="5" UUID="0d42284f-9e9b-4a81-a88b-8bb42bc0a905" NAME="Id Pool Filled from Remote" IDENTIFIER_TYPE="4" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
+	<IDGEN_IDENTIFIER_SOURCE ID="6" UUID="0d42284f-9e9b-4a81-a88b-8bb42bc0a906" NAME="Remote to fill Id Pool" IDENTIFIER_TYPE="4" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
+
+	<IDGEN_SEQ_ID_GEN ID="1" first_identifier_base="1" NEXT_SEQUENCE_VALUE="6" base_character_set="ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789" PREFIX="" min_length="0" max_length="0"/>
+
+	<IDGEN_REMOTE_SOURCE ID="2" URL="http://localhost:8080/openmrs/module/idgen/getIdentifier"/>
+	<IDGEN_REMOTE_SOURCE ID="6" URL="http://localhost:8080/openmrs/module/idgen/getIdentifier"/>
+	
+	<IDGEN_ID_POOL ID="3" SOURCE="1" BATCH_SIZE="1000" MIN_POOL_SIZE="100" sequential="1" refill_with_scheduled_task="0"/>
+    <IDGEN_ID_POOL ID="4" SOURCE="1" BATCH_SIZE="1000" MIN_POOL_SIZE="1" sequential="0" refill_with_scheduled_task="1"/>
+	<IDGEN_ID_POOL ID="5" SOURCE="2" BATCH_SIZE="3" MIN_POOL_SIZE="4" sequential="1" refill_with_scheduled_task="0"/>
+
+	<IDGEN_POOLED_IDENTIFIER ID="1" POOL_ID="3"  IDENTIFIER="00001" UUID="1d47284f-9e9b-4a81-a88b-8bb42bc0a901"/>
+	<IDGEN_POOLED_IDENTIFIER ID="2" POOL_ID="3"  IDENTIFIER="00002" UUID="2d47284f-9e9b-4a81-a88b-8bb42bc0a901"/>
+	<IDGEN_POOLED_IDENTIFIER ID="3" POOL_ID="3"  IDENTIFIER="00003" UUID="3d47284f-9e9b-4a81-a88b-8bb42bc0a901"/>
+	<IDGEN_POOLED_IDENTIFIER ID="4" POOL_ID="3"  IDENTIFIER="00004" UUID="4d47284f-9e9b-4a81-a88b-8bb42bc0a901"/>
+	<IDGEN_POOLED_IDENTIFIER ID="5" POOL_ID="3"  IDENTIFIER="00005" UUID="5d47284f-9e9b-4a81-a88b-8bb42bc0a901"/>
+
+    <IDGEN_AUTO_GENERATION_OPTION ID="1" SOURCE="1" IDENTIFIER_TYPE="1" MANUAL_ENTRY_ENABLED="1" AUTOMATIC_GENERATION_ENABLED="1" />
+    <IDGEN_AUTO_GENERATION_OPTION ID="2" SOURCE="2" IDENTIFIER_TYPE="2" LOCATION="1" MANUAL_ENTRY_ENABLED="1" AUTOMATIC_GENERATION_ENABLED="1" />
+    <IDGEN_AUTO_GENERATION_OPTION ID="3" SOURCE="3" IDENTIFIER_TYPE="2" LOCATION="2" MANUAL_ENTRY_ENABLED="1" AUTOMATIC_GENERATION_ENABLED="1" />
+	<IDGEN_AUTO_GENERATION_OPTION ID="4" SOURCE="5" IDENTIFIER_TYPE="4" LOCATION="2" MANUAL_ENTRY_ENABLED="0" AUTOMATIC_GENERATION_ENABLED="1" />
+
```

Comment:
Can we reduce this space to just one line?

Reply:
Fixed